browser/net/anon_http_auth_preferences.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 The Brave Authors. All rights reserved.
```

Comment:
nit: 2019 not 2018.
And all other files we created uses below.
```
/* Copyright (c) 2019 The Brave Authors. All rights reserved.
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Reply:
Whoops, copypasta from another file; lint didn't catch it.  Fix pushed.